src/libs/events.js

```diff
@@ -0,0 +1,8 @@
+const eventList = {
+  userRegister: 'user:register',
+  userSignIn: 'user:signin',
+
+  pageView: 'page:view',
```

Comment:
Are the pageView hooks to be implemented in a future ticket?

Reply:
Yeah, pageviews should be included in future tickets.